lib/grover.rb

```diff
@@ -23,7 +23,7 @@ class Processor < Schmooze::Base
     dependencies puppeteer: 'puppeteer'
 
     def self.launch_params
-      ENV['CI'] == 'true' ? "{args: ['--no-sandbox', '--disable-setuid-sandbox']}" : '{}'
+      ENV['GROVER_NO_SANDBOX'] == 'true' ? "{args: ['--no-sandbox', '--disable-setuid-sandbox']}" : '{}'
```

Comment:
Hmm.. interesting that Travis no longer needs this to be set! Spec tests pass.. fine with me